virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc

```diff
@@ -10,9 +10,9 @@ By default, {VirtProductName} is installed with a single, internal pod network.
 
 You can create a Linux bridge network and attach a virtual machine (VM) to the network by performing the following steps:
 
-. xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-creating-linux-bridge-nncp_virt-connecting-vm-to-linux-bridge[Create a Linux bridge node network configuration policy (NNCP)].
```

Comment:
This formatting is not supported. The xref links must follow the pattern outlined in the style guide or they will break on the customer portal.

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references